Dockerfile

```diff
@@ -1,79 +1,23 @@
-# Use buster image because the default bullseye image has updated coreutils that require a newer
-# linux kernel than provided by CircleCI, per
-# https://forums.docker.com/t/multiple-projects-stopped-building-on-docker-hub-operation-not-permitted/92570/6
-# and https://forums.docker.com/t/multiple-projects-stopped-building-on-docker-hub-operation-not-permitted/92570/11
-FROM python:3.8.12-slim-buster
-MAINTAINER Harold Woo <hwoo@mozilla.com>
+FROM apache/airflow:slim-2.3.3-python3.8
 
-ARG AIRFLOW_UID=10001
```

Comment:
This may cause issues with dags on the file mount in the container, we need to ensure airflow can read the dags after this change